internal/queue/loops/fake/fake_test.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright (c) 2025 Diagrid Inc.
+Licensed under the MIT License.
+*/
+
+package fake
+
+import (
+	"testing"
+
+	"github.com/diagridio/go-etcd-cron/internal/queue/loops"
+)
+
+func Test_Fake(t *testing.T) {
+	var _ loops.Interface[int] = New[int]()
```

Comment:
why do we need this? couldn't we do this type check in fake.go?

Reply:
A style thing.. this is how this is tested everywhere else